src/adaptors/notional-v3/index.js

```diff
@@ -5,13 +5,15 @@ const { default: BigNumber } = require('bignumber.js');
 
 const API = (chain) =>
   chain === 'ethereum'
-    ? `https://data-dev.notional.finance/mainnet/yields`
-    : `https://data-dev.notional.finance/${chain}/yields`;
+    ? `https://registry.notional.finance/mainnet/yields`
+    : `https://registry.notional.finance/${chain}/yields`;
 const NOTE_Mainnet = '0xCFEAead4947f0705A14ec42aC3D44129E1Ef3eD5';
 
 const SUBGRAPHS = {
+  // arbitrum graph id: 'DnghsCNvJ4xmp4czX8Qn7UpkJ8HyHjy7cFN4wcH91Nrx',
   arbitrum:
     'https://api.studio.thegraph.com/query/60626/notional-v3-arbitrum/version/latest',
```

Comment:
just change this to `sdk.graph.modifyEndpoint(
  'DnghsCNvJ4xmp4czX8Qn7UpkJ8HyHjy7cFN4wcH91Nrx'
)`

Reply:
@slasher125 made the change, the test runs fine locally when I use my own GRAPH_API_KEY but guess it's failing in CI on a missing key